man/en/sos-collect.1

```diff
@@ -0,0 +1,312 @@
+.TP
+\fB\-\-master\fR MASTER
+Specify a master node for the cluster.
+
+If provided, then sos collect will check the master node, not localhost, for determining
+the type of cluster in use.
+.TP
+\fB\-o\fR ONLY_PLUGINS, \fB\-\-only\-plugins\fR ONLY_PLUGINS
+Sosreport option. Run ONLY the plugins listed.
+
+Note that a cluster profile will NOT override this option. This may cause the sosreports
+generated to not contain the relevant output for a given type of cluster.
+
+This option supports providing a comma-delimited list of plugins.
+.TP
+\fB\-\-password\fR
+Specifying this option will cause sos collect to prompt the user for an SSH password
+that will be used to connect to all nodes.
+
+If you have differing passwords for the same user across cluster nodes, you should
+ideally deploy SSH keys, but the \-\-password\-per\-node option is also available.
+.TP
+\fB\-\-password\-per\-node\fR
+When using this option, sos collect will prompt the user for the SSH password for
+each node that will have an sosreport collected from it individually before attempting
+to connect to the nodes.
+.TP
+\fB\-\-preset\fR PRESET
+Specify a sos preset to use, note that this requires sos-3.6 or later to be installed
+on the node. The given preset must also exist on the remote node - local presets
+are not used.
+
+If \fB\-\-preset\fR is specified and a given node either does not have that preset
+defined, or has a version of sos prior to 3.6, this option is ignored for that node.
+.TP
+\fB\-p\fR SSH_PORT, \fB\-\-ssh\-port\fR SSH_PORT
+Specify SSH port for all nodes. Use this if SSH runs on any port other than 22.
+.TP
+\fB\-\-ssh\-user\fR SSH_USER
+Specify an SSH user for sos collect to connect to nodes with. Default is root.
+
+sos collect will prompt for a sudo password for non-root users.
+.TP
+\fB\-s\fR SYSROOT, \fB\-\-sysroot\fR SYSROOT
+Sosreport option. Specify an alternate root file system path.
+.TP
+\fB\-\-sos-cmd\fR SOS_CMD
+Define all options that sosreport should be run with on the nodes. This will
+override any other commandline options as well as any options specified by a 
+cluster profile.
+
+The sosreport command will execute as 'sosreport --batch SOS_CMD'. The BATCH 
+option cannot be removed from the sosreport command as it is required to run 
+sosreport non-interactively for sos collect to function.
+.TP
+\fB\-t\fR THREADS \fB\-\-threads\fR THREADS
+Specify the number of threads to use for concurrent collection of sosreports.
+
+If the number of nodes enumerated exceeds the number of threads, then sos collect
+will start collecting from the first X number of nodes and then continue to iterate
+through the remaining nodes as sosreport collection finishes.
+
+Defaults to 4.
+.TP
+\fB\-\-timeout\fR TIMEOUT
+Timeout for sosreport generation on each node, in seconds.
+
+Note that sosreports are collected in parallel, so this can also be considered to be
+approximately the same as a timeout for the entire collection process. 
```

Comment:
This statement ignores the impact of `--threads`. Defaulting to 4, when I collect `sosreport` from e.g. 8 systems, I can need to wait up to `2*timeout` seconds.

Reply:
Yup, need to reword that.